swarm/storage/localstore/mode_put_test.go

```diff
@@ -235,10 +235,13 @@ func TestModePut_sameChunk(t *testing.T) {
 			defer cleanupFunc()
 
 			for i := 0; i < 10; i++ {
-				err := db.Put(context.Background(), tc.mode, ch)
+				exists, err := db.Put(context.Background(), tc.mode, ch)
 				if err != nil {
 					t.Fatal(err)
 				}
+				if i == 0 && exists {
```

Comment:
This condition is testing only when the chunk does not exist at the first put, but there is no test for conditions that the chunk exists in the following puts.

Reply:
oops :) now better?